### ISSUE TITLE
Definition of sequence/@preserveOrder is backwards

### ISSUE THREAD

**Issue by hcayless** [TEIC/TEI#2390]
In https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-sequence.html, the definition of `@preserveOrder` is

> if true, indicates that the order in which component elements of a sequence appear in a document must correspond to the order in which they are given in the content model.

It should read something like: "If false, indicates that the component elements of a sequence may occur in any order."

When preserveOrder is false, in a RelaxNG schema generated from the ODD, an `<interleave>` element is produced, meaning children may occur in any order, rather than needing to have the order given in the sequence.

tl;dr: `sequence/@preserveOrder="true"` doesn't do anything, it's `sequence/@preserveOrder="false"` that matters, and so that should be the focus of the definition.

**Comment by joeytakeda** [comment]
Possibly a duplicate of  #2154 ?

**Comment by hcayless** [comment]
@joeytakeda It is indeed! That one seems to have gotten wrapped around the axle somewhere along the line...
